doc/Command-Reference.md

```diff
@@ -4336,6 +4356,29 @@ MPLS operation for either physical, portchannel, or VLAN interface can be config
   admin@sonic:~$ sudo config interface mpls remove Ethernet4
   ```
 
+**config interface loopback-action <interface_name> <action> (Versions >= 202205)**
+
+This command is used for setting the action being taken on packets that ingress and get routed on the same IP interface.
+Loopback action can be set on IP interface from type physical, portchannel, VLAN interface and VLAN subinterface.
+Loopback action can be drop or forward.
+
+- Usage:
+  ```
+  config interface loopback-action --help
+  Usage: config interface loopback-action [OPTIONS] <interface_name> <action>
+
+    Set IP interface loopback action
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ config interface loopback-action Ethernet0 drop
```

Comment:
Since 'show' is part of 'ip' sub-command, should we have this as 'config interface ip loopback-action ...'

Reply:
Done. I am closing his PR, those changes will be part of PR https://github.com/Azure/sonic-utilities/pull/2192